redbot/core/core_commands.py

```diff
@@ -3633,8 +3633,22 @@ async def _set_api(
         else:
             if ctx.bot_permissions.manage_messages:
                 await ctx.message.delete()
+            embed = discord.Embed()
+            for api_service_name, token in tokens.items():
+                if token.startswith("<") and token.endswith(">"):
+                    angle_bracket_warning = (
+                        "You may have failed to properly format your {api_service_name}. If you were told to enter a key"
+                        " with an example such as [p]set api {service} api_key <your_api_key_here>, and your API key"
+                        " was HREDFGWE, make sure to run [p]set api {service} api_key HREDFGWE, and not "
+                        " [p]set api {service} api_key <HREDFGWE>"
```

Comment:
Wrapping the command examples with inline marks may be appropriate to improve readability.

```suggestion
                        "You may have failed to properly format your {api_service_name}. If you were told to enter a key"
                        " with an example such as `[p]set api {service} api_key <your_api_key_here>`, and your API key"
                        " was `HREDFGWE`, make sure to run `[p]set api {service} api_key HREDFGWE` and not "
                        " `[p]set api {service} api_key <HREDFGWE>`."
```